app/models.py

```diff
@@ -130,10 +130,6 @@ def get_dict(self):
         data.pop('password', None)
         if self.username:
             data['username'] = self.username.lower()
-        try:
```

Comment:
I don’t want to implement it here because then user emails will be revealed in other cases, like when fetching a users profile. Instead we should just explicitly add it in the /me endpoint I think

Reply:
Fair enough.